### ISSUE TITLE
PDF title shows up correctly locally, but not in production

### ISSUE THREAD

**Issue by andrewmcodes** [mileszs/wicked_pdf#752]
I am having an issue that I hope someone has some knowledge about. I am using wicked_pdf to allow a user to download a report. I am running into an issue where when the pdf opens in your browser, the title is set as the index number of the pdf (e.g. 28.pdf). When I download the form, the file name is correct. However, **this issue does not happen locally**, only when the pdf is delivered by the server. 

I have tried multiple things and still cannot figure this out. In my example.pdf.haml file, I am using this:
```
!!!
%html
  %head
    %title Separation Summmary
    = wicked_pdf_stylesheet_link_tag "separation_form_pdf"
  %body
   ......
```
and have tried multiple variations of that like not including the top two lines. 

Here is the relevant part of my controller:
```
respond_to do |format|
          format.pdf do
            render pdf: "Example",
                   title: "Example"
          end
          format.html do
......
```
If anyone has suggestions, I would appreciate it.

**Comment by unixmonkey** [comment]
Looks like you cross-posted this to [StackOverflow here](https://stackoverflow.com/questions/51293428/wicked-pdf-title-shows-correctly-locally-but-not-on-the-server). Adding a link here for posterity.

**Comment by andrewmcodes** [comment]
You are correct. The answer on stackoverflow provided a bandaid, but not a true fix. Going with that option for now but still looking for a true fix.
